### ISSUE TITLE
Jacoco error

### ISSUE THREAD

**Issue by saiqulhaq** [rollbar/rollbar-android#46]
```
Caused by java.lang.ClassNotFoundException
Didn't find class "org.jacoco.agent.rt.internal_8ff85ea.Offline" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/com.hb.hungryhub-tRCOmkHIbuGaAy01iEe1oQ==/base.apk"],nativeLibraryDirectories=[/data/app/com.hb.hungryhub-tRCOmkHIbuGaAy01iEe1oQ==/lib/arm64, /data/app/com.hb.hungryhub-tRCOmkHIbuGaAy01iEe1oQ==/base.apk!/lib/arm64-v8a, /system/lib64, /product/lib64]]
```

```
Fatal Exception: java.lang.NoClassDefFoundError: Failed resolution of: Lorg/jacoco/agent/rt/internal_8ff85ea/Offline;
       at com.rollbar.android.Rollbar.$jacocoInit(:13)
       at com.rollbar.android.Rollbar.init()
       at com.hb.hungryhub.application.App.onCreate(App.java:44)
       at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1162)
       at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6717)
       at android.app.ActivityThread.access$2000(ActivityThread.java:273)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2020)
       at android.os.Handler.dispatchMessage(Handler.java:112)
       at android.os.Looper.loop(Looper.java:216)
       at android.app.ActivityThread.main(ActivityThread.java:7625)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
```

we never have this issue previously, just call ` Rollbar.init(this);`

**Comment by devitito** [comment]
Same here.

**Comment by basoko** [comment]
Sorry for this issue, we have marked that version broken because we had also some problems when publishing it.
We have published a 1.7.1 that should work https://github.com/rollbar/rollbar-java/releases/tag/v1.7.1.
